src/confirmation-dialog/ConfirmationDialog.tsx

```diff
@@ -16,8 +17,10 @@ export interface ConfirmationDialogProps extends Partial<DialogProps> {
     description?: string | ReactNode;
     onSave?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
     onCancel?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+    onInfoAction?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
     saveText?: string;
     cancelText?: string;
+    infoActionText?: string;
```

Comment:
Generic note/comment for future components. Instead of pairs `onSomething`, `somethingText` we can have a unified object. Now that we have TS, the user can readily see what must be passed. For example: `something: {onClick: ..., text?: string}`. Drawback: we have to memoize it somehow, as it as an object.

Reply:
Yeah, I guess here it would make sense to use ``useRef`` to store it and update it programmatically on each prop change but holding the ref intact so it doesn't change the object signature. It's probably the pattern I dislike the most of the react functional components...

Reply:
Or `useMemo`. On this particular point, the original sin comes from JS: `{} === {} #=> false`